WordPress/Classes/ViewRelated/Comments/Content/CommentDetailContentView.swift

```diff
@@ -0,0 +1,82 @@
+import UIKit
+
+final class CommentDetailContentView: UIView {
```

Comment:
As I've mentioned in the PR description, most of the logic was extracted from `CommentContentTableViewCell`, except:
- The cell was using a `NSLayoutConstraint` to constraint the height, I'm using `intrinsicContentSize` instead.
- The cell was giving the 2 methods to render comments ( `web`, `richContent` ). I've extracted only the `web` rendering method because it was the one used in `Comment Detail` screen.
- Added a `Configuration` type to encapsulate configuration properties.